srcpkgs/f3d/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'f3d'
+pkgname=f3d
+version=2.1.0
+revision=1
+build_style=cmake
+makedepends="vtk-devel"
+depends="vtk"
+short_desc="Fast and minimalist 3D viewer"
+maintainer="Jipok <void@jipok.ru>"
+license="BSD-3-Clause"
+homepage="https://f3d.app"
+changelog="https://f3d.app/doc/CHANGELOG.html"
+distfiles="https://github.com/f3d-app/f3d/archive/refs/tags/v${version}.tar.gz"
+checksum=3e5e6c2c16da4d7ccce8b6e316ab8007592a2bc0fc11a513f1ebac8c7f0f95d2
+
+post_install() {
+	vlicense LICENSE.md
+}
+
+do_configure() {
+	echo AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+	exit 1
```

Comment:
This looks wrong

Reply:
To be honest, I rechecked this small template and did not see any problems in it. Damn, this is so stupid and embarrassing.